CRM/Upgrade/Incremental/Base.php

```diff
@@ -585,4 +585,26 @@ public static function rebuildMultilingalSchema($ctx, $version = NULL) {
     return TRUE;
   }
 
+  public static function alterColumn($ctx, $table, $column, $properties, $localizable = FALSE): bool {
+    $locales = CRM_Core_I18n::getMultilingual();
+    $queries = [];
+    if ($localizable) {
+      if ($locales) {
```

Comment:
These two if statements could be combined to 
```suggestion
    if ($localizable && $locales) {
```